README.md

```diff
@@ -53,7 +52,7 @@ We can run pretty much any code in REPL. Let's do some math:
 1
 ```
 
-Now let's define an expression (function which returns a value) which take two parameters and returns the sum of them. 
+Now let's define an expression (function which returns a value) which take two parameters and returns the sum of them.
```

Comment:
You can cut some of the JS explanations because students will have already covered those topics. They should understand by this point the difference between declaring a function and invoking the function